examples/MonteCarloExamples/scenarioAnalyzeMonteCarlo.py

```diff
@@ -68,11 +68,11 @@
 #. Install this particular version of ``panel`` package first.  It must be done alone as it upgrades
    ``bokeh`` to a version that is too new::
 
-        pip3 install --upgrade panel==0.9.7
+        pip3 install --upgrade panel==1.4.4
 
 #. Next, install the following particular python package versions::
 
-        pip3 install --upgrade bokeh==1.2.0 holoviews==1.12.3 param==1.9.3 hvplot==0.6.0
+        pip3 install --upgrade bokeh==3.4.2 holoviews==1.12.3 param==2.1.1 hvplot==0.10.0
```

Comment:
Should I create a PR in ema-gnc updating requirements.txt file?

Reply:
Yes